lib/Default/MySqlProperties.class.php

```diff
@@ -0,0 +1,49 @@
+<?php declare(strict_types=1);
+
+use Dotenv\Dotenv;
+
+class MySqlProperties {
+	private const CONFIG_MYSQL_HOST = "MYSQL_HOST";
+	private const CONFIG_MYSQL_USER = "MYSQL_USER";
+	private const CONFIG_MYSQL_PASSWORD = "MYSQL_PASSWORD";
+	private const CONFIG_MYSQL_DATABASE = "MYSQL_DATABASE";
+	private string $host;
+	private string $user;
+	private string $password;
+	private string $databaseName;
+
+	public function __construct() {
+		$config = Dotenv::createArrayBacked(ROOT);
+		$array = $config->load();
+		self::validateConfig($config);
+		[
+			MySqlProperties::CONFIG_MYSQL_HOST => $this->host,
```

Comment:
Okay, I think this is my last suggestion: Should we use `self::CONFIG_MYSQL_*` (instead of explicitly writing out the class name here)?